packages/core/src/options.ts

```diff
@@ -79,6 +79,8 @@ export interface WriteOptions extends WriteRetryOptions {
   defaultTags?: Record<string, string>
   /** HTTP headers that will be sent with every write request */
   headers?: {[key: string]: string}
+  /** When specified, write bodies larger than the threshold are gzipped  */
+  gzipThreshold?: number
```

Comment:
What do you think about disabling the GZIP compression by setting `gzipThreshold` to `-1`?

Reply:
It would be an extra case to handle that I generally prefer to avoid. It can be now disabled with `{gzipThreshold: undefined}`. An intuitive Number.MAX_SAFE_INTEGER` or `Number.MAX_VALUE` will do the same job.

Reply:
ok